modules/exploits/linux/http/zyxel_parse_config_rce.rb

```diff
@@ -0,0 +1,157 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Zyxel parse_config.py Command Injection',
+        'Description' => %q{
+          This module exploits vulnerabilities in multiple Zyxel devices including the VPN, USG and APT series.
+          The affected firmware versions depend on the device module, see this module's documentation for more details.
+        },
+        'Author' => [
+          'SSD Secure Disclosure technical team', # discovery
+          'jheysel-r7',    # Msf module
+          'Jacob Baines',  # Testing
+        ],
+        'References' => [
+          [ 'URL', 'https://ssd-disclosure.com/ssd-advisory-zyxel-vpn-series-pre-auth-remote-command-execution/'],
+          [ 'CVE', '2023-33012']
+        ],
+        'License' => MSF_LICENSE,
+        'Platform' => ['linux', 'unix'],
+        'Privileged' => true,
+        'Arch' => [ ARCH_CMD ],
+        'Targets' => [
+          [ 'Automatic Target', {}]
+        ],
+        'DefaultTarget' => 0,
+        'DisclosureDate' => '2024-01-24',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE, ],
+          'SideEffects' => [ ARTIFACTS_ON_DISK, CONFIG_CHANGES ],
+          'Reliability' => [ ] # This vulnerability can only be exploited once, more info: https://vulncheck.com/blog/zyxel-cve-2023-33012#you-get-one-shot
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('WRITABLE_DIR', [ true, 'A directory where we can write files', '/tmp' ]),
```

Comment:
I'm wondering if this option should be added to the documentation. `WRITABLE_DIR` doesn't seem to be part of the `msftidy_docs.rb` universal option [list](https://github.com/rapid7/metasploit-framework/blob/ddaa21d8c3fc097350100900f14354db84715e92/tools/dev/msftidy_docs.rb#L248)

Reply:
Good call. I've added this to the docs in: [12b1936](https://github.com/rapid7/metasploit-framework/pull/19204/commits/12b1936e16aaa883ee786b97495f960756d407d9)